nextflow.config

```diff
@@ -181,6 +181,7 @@ profiles {
         apptainer.enabled       = false
     }
     docker {
+        // includeConfig 'conf/containers/containers_docker_amd64.config'
```

Comment:
Include at the top, above profiles? That way it's the default for anyone not using profiles (eg. someone wanting to use Podman, or whatever).

Reply:
I think we still need to solve ![image](https://github.com/user-attachments/assets/24298a2b-6209-4875-8751-a7488951498f)